modules/mobile_session.lua

```diff
@@ -148,12 +155,18 @@ function mt.__index:Send(message)
         binaryData = message.binaryData
       }
     })
-  if not self.cor_id_func_map[self.correlationId] then
-    for fname, fid in pairs(functionId) do
-      if fid == message.rpcFunctionId then
-        self.cor_id_func_map[self.correlationId] = fname
-        break
+  -- During StartSession Send called with no correllation id
+  if message_correlation_id then
```

Comment:
Are this comment and check actual? Because if there is no rpcCorrelationId, we set `message_correlation_id` equals `self.correlationId + 1` (line 137)